### ISSUE TITLE
Link to entries from a different site

### ISSUE THREAD

**Issue by luke-nehemedia** [sebastian-lenz/craft-linkfield#260]
I use this plugin on a couple of multi-site installations. I can link to one of those sites, but I cannot link to entries from other sites. 

Would it be possible to enable a site-switcher, as it is implemented in craft's own entry-selection dialog?

**Comment by luke-nehemedia** [comment]
I guess this option is only available in Craft4?

**Comment by sebastian-lenz** [comment]
Yes, you need the latest Craft and plugin versions for this option.
